README.md

```diff
@@ -52,3 +52,7 @@ First change models.py and then run `python manage.py makemigrations` to automat
 2. If you want to access the postgresql database,
 
 Run `python manage.py dbshell`.
+
+3. If you want to access the backend admin site on http://127.0.0.1:8000/admin/ but without a user
+
+Run `python manage.py createsuperuser` to create a super user.
```

Comment:
What method are we going to use for creating user accounts on staging and production?

Reply:
This is a question that I wanted to ask and discuss with you, or/and infrastructure team?